### PR TITLE
fix clang-tidy warnings

### PR DESCRIPTION
**Pull request by radexx99**
https://matterfi.atlassian.net/wiki/spaces/ME/pages/34209793/Open+Transactions+Pull+Request+Summaries#PR:-#1899-add-clang-tidy-to-actions

look PR: 1899

Sorry for error, this is 2nd commit of PR1899

### CODE REVIEW

**Comment by radexx99** [comment]
It is second commit of PR 1899 sorry for error

**Comment by wiesiekpap** [comment]
What is the significance of the PR#1900 note linked to this? I interpreted that as a summary of the changes, but many of the files mentioned in the note cannot even be found here. Specifically, I have looked for Secret.cpp and StateMachine.hpp, because according to the note they should include meaningful change. Can you please clarify?

**Review by wiesiekpap — CHANGES_REQUESTED** [review]
A clarification is needed as to the content. I cannot find the meaningful changes mentioned in PR#1900 note the description of this PR lines to.

File: tests/blockchain/Test_BlockchainAPI.cpp [hunk 1]
```diff
@@ -548,8 +548,6 @@ TEST_F(Test_BlockchainAPI, invalid_nym)
     EXPECT_EQ(list.size(), 0);
     EXPECT_EQ(list.count(accountID), 0);
 
-    loaded = false;
```
Comment (wiesiekpap) [inline tests/blockchain/Test_BlockchainAPI.cpp:551]: EXPECT_FALSE does not stop the test, so assuming loaded to be false here is wrong. Please explain or revert the change.

**Review by wiesiekpap — CHANGES_REQUESTED** [review]
The change in  tests/blockchain/Test_BlockchainAPI.cpp needs clang-tidy exemption, because it is wrong. Alternatively, get around it by introducing a new boolean for the other check.

File: tests/blockchain/Test_BlockchainAPI.cpp (hunk 1 above)
Comment (radexx99) [reply tests/blockchain/Test_BlockchainAPI.cpp:551]: good catch i will create another boolean for this

**Comment by radexx99** [comment]
this PR would be postponed to not defined future
